cmd/collector/Dockerfile

```diff
@@ -7,4 +7,4 @@ COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certifica
 
 EXPOSE 14267
 COPY collector-linux /go/bin/
-ENTRYPOINT ["/go/bin/collector-linux"]
+ENTRYPOINT ["/go/bin/collector-linux", "--cassandra.keyspace=jaeger_v1_dc1", "--cassandra.servers=cassandra", "--collector.zipkin.http-port=9411"]
```

Comment:
We might need another solution. I pulled this branch and tried to build and run the collector container with non-cassandra memory store:
```
wjang:jaeger won$ docker run -e SPAN_STORAGE_TYPE=memory jaegertracing/jaeger-collector:latest
unknown flag: --cassandra.keyspace
Error: unknown flag: --cassandra.keyspace
```

Reply:
OK I see the problem. The solution is probably using the original `command` override but adding the program as the first argument.